cmd/tetra/tracingpolicy/tracingpolicy.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/cilium/tetragon/api/v1/tetragon"
 	"github.com/cilium/tetragon/cmd/tetra/common"
 	"github.com/cilium/tetragon/pkg/btf"
+	"github.com/cilium/tetragon/pkg/logger"
```

Comment:
nit: This seems to be related to the next commit, not this one.

Reply:
thanks!